src/install/common.py

```diff
@@ -46,28 +47,25 @@ def main(distribution):  # pylint: disable=too-many-statements
         pip3_install_packages('testresources')
 
     # get a bug free recent pip version
-    if distribution == 'fedora':
-        dnf_remove_packages('python3-pip', 'python3-setuptools', 'python3-wheel')
-    else:
-        apt_remove_packages('python3-pip', 'python3-setuptools', 'python3-wheel')
+    if not is_virtualenv():
+        if distribution == 'fedora':
+            dnf_remove_packages('python3-pip', 'python3-setuptools', 'python3-wheel')
+        else:
+            apt_remove_packages('python3-pip', 'python3-setuptools', 'python3-wheel')
 
-    install_pip('python3')
+    install_pip()
     pip3_install_packages('setuptools')
 
     # install general python dependencies
     if distribution == 'fedora':
         dnf_install_packages('file-devel')
         dnf_install_packages('libffi-devel')
-        dnf_install_packages('python3-tlsh')
-        dnf_install_packages('python3-ssdeep')
     else:
         apt_install_packages('libmagic-dev')
         apt_install_packages('libfuzzy-dev')
-        apt_install_packages('python3-tlsh')
-        pip3_install_packages('ssdeep')
 
     pip3_install_packages('git+https://github.com/fkie-cad/fact_helper_file.git')
-    pip3_install_packages('psutil', 'pytest', 'pytest-cov', 'pylint', 'python-magic', 'xmltodict', 'yara-python==3.7.0', 'appdirs', 'flaky')
+    pip3_install_packages('psutil', 'pytest', 'pytest-cov', 'pylint', 'python-magic', 'xmltodict', 'yara-python==3.7.0', 'appdirs', 'flaky', 'python-tlsh', 'ssdeep')
```

Comment:
No wonder this breaks on fedora. Why was ssdeep moved out of the conditional block?

Reply:
I was under the impression that it worked in  #618. Surely, I have been mistaken. -> back to the conditional block